diff --git a/cmd/tools/Makefile.am b/cmd/tools/Makefile.am
index c1764e1..deafb51 100644
--- a/cmd/tools/Makefile.am
+++ b/cmd/tools/Makefile.am
@@ -122,7 +122,8 @@ gvcolor_SOURCES = gvcolor.c colxlate.c colortbl.h
 
 gvcolor_LDADD = \
 	$(top_builddir)/lib/ingraphs/libingraphs_C.la \
-	$(top_builddir)/lib/cgraph/libcgraph.la
+	$(top_builddir)/lib/cgraph/libcgraph.la \
+	-lm
 
 
 bcomps_SOURCES = bcomps.c
