--- graphviz-stable_release_2.42.2/tclpkg/gv/Makefile.am	2019-09-12 03:11:30.000000000 +0100
+++ graphviz-stable_release_2.42.2.new/tclpkg/gv/Makefile.am	2019-12-06 13:05:44.064150008 +0000
@@ -130,7 +130,11 @@
 libgv_ocaml_la_SOURCES = $(BASESOURCES) gv_dummy_init.c
 libgv_ocaml_la_LIBADD = $(BASELIBS) $(OCAML_LIBS)
 libgv_ocaml_la_LDFLAGS = -module -avoid-version
-libgv_ocaml_la_CPPFLAGS = $(BASECPPFLAGS) $(OCAML_INCLUDES) -Dint64=int64_t
+libgv_ocaml_la_CPPFLAGS = \
+	$(BASECPPFLAGS) \
+	$(OCAML_INCLUDES) \
+	-Dint64=int64_t \
+	-fpermissive
 gv.ml gv.mli: gv_ocaml.cpp
 gv_ocaml.cpp: gv.i
 	$(SWIG) -c++ -ocaml -o gv_ocaml.cpp $(srcdir)/gv.i
